test/integration/tests/style_tests.json

```diff
@@ -1258,202 +1258,202 @@
             "refs": [
-                "0910e0d"
+                "aabb1c2"
             ]
         },
         "plot/xAxis/ticks/length/number/static_10": {
             "refs": [
-                "7892e9d"
+                "aabb1c2"
             ]
         },
         "plot/xAxis/ticks/length/number/static_50": {
             "refs": [
-                "f47bfc1"
+                "aabb1c2"
             ]
         },
         "plot/xAxis/ticks/length/pixel/animated_1-50": {
             "refs": [
-                "cdfe7d0"
+                "53fd4ec"
             ]
         },
         "plot/xAxis/ticks/length/pixel/animated_50-1": {
             "refs": [
-                "6307e34"
+                "53fd4ec"
             ]
         },
         "plot/xAxis/ticks/length/pixel/static_1": {
             "refs": [
-                "0910e0d"
+                "aabb1c2"
             ]
         },
         "plot/xAxis/ticks/length/pixel/static_10": {
             "refs": [
-                "7892e9d"
+                "aabb1c2"
             ]
         },
         "plot/xAxis/ticks/length/pixel/static_50": {
             "refs": [
-                "f47bfc1"
+                "aabb1c2"
             ]
         },
         "plot/xAxis/ticks/lineWidth/animated_1-100": {
             "refs": [
-                "827e1c4"
+                "53fd4ec"
             ]
         },
         "plot/xAxis/ticks/lineWidth/animated_100-1": {
             "refs": [
-                "262bea0"
+                "53fd4ec"
             ]
         },
         "plot/xAxis/ticks/lineWidth/static_1": {
             "refs": [
-                "567dbdd"
+                "aabb1c2"
             ]
         },
         "plot/xAxis/ticks/lineWidth/static_10": {
             "refs": [
-                "9357126"
+                "aabb1c2"
             ]
         },
         "plot/xAxis/ticks/lineWidth/static_100": {
             "refs": [
-                "863c0ca"
+                "aabb1c2"
             ]
         },
         "plot/xAxis/ticks/position/animated_center-inside": {
             "refs": [
-                "c2a09e6"
+                "53fd4ec"
             ]
         },
         "plot/xAxis/ticks/position/animated_center-outside": {
             "refs": [
-                "876e456"
+                "53fd4ec"
             ]
         },
         "plot/xAxis/ticks/position/animated_inside-center": {
             "refs": [
-                "63dee9d"
+                "53fd4ec"
             ]
         },
         "plot/xAxis/ticks/position/animated_inside-outside": {
             "refs": [
-                "4b4f2c9"
+                "53fd4ec"
             ]
         },
         "plot/xAxis/ticks/position/animated_outside-center": {
             "refs": [
-                "4e7fdb8"
+                "53fd4ec"
             ]
         },
         "plot/xAxis/ticks/position/animated_outside-inside": {
             "refs": [
-                "9cd88ee"
+                "53fd4ec"
```

Comment:
Same hash for these tests?

Reply:
Background bars orientation changed, the ticks moved to the yAxis
![Screenshot from 2023-07-17 11-50-48](https://github.com/vizzuhq/vizzu-lib/assets/61405792/84a7db7c-9431-48dc-8716-41bf6ab703b7)